lib/ClustersBackend/PlacesBackend.php

```diff
@@ -162,7 +162,7 @@ public function getClustersInternal(int $fileid = 0): array
         $lang = Util::getUserLang();
         foreach ($places as &$row) {
             $row['osm_id'] = (int) $row['osm_id'];
-            $row['count'] = $marked ? 0 : (int) $row['count']; // the count is incorrect
+            $row['count'] = (int) $row['sub.count'];
```

Comment:
`$row['count']` not `sub.count`

Reply:
You'r right, I thougth i tested it witch just count and it didn't work, but it works now. I must have made a wrong test. It should be fixed now